torq.sh

```diff
@@ -234,19 +233,27 @@ else
 fi
 
 if [[ "$1" == "start" ]]; then
-  if [[ $(echo ${BASH_ARGV[*]} | grep -e print) ]]; then         
-    checkextrascsv "${*%${!#}}";
-    for p in $PROCS;
-    do 
-      print "$p";  
-    done
-  else 
-    checkextrascsv "$*";
-    startprocs "$PROCS";
-  fi
+  checkextrascsv "$*";
+  startprocs "$PROCS";
+elif [[ "$1" == "print" ]]; then         
+  checkextrascsv "$*";
+  for p in $PROCS;
+  do 
+    print "$p";  
+  done
 elif [[ "$1" == "stop" ]]; then
   checkextrascsv $@;
   stopprocs "$PROCS";
+elif [[ "$1" == "debug" ]]; then
+  checkextrascsv "$*";
+  if [[ $(echo $PROCS | wc -w) -gt 1 ]]; then 
+    echo "ERROR: Cannot debug more than one process at a time"
+  else 
+    for p in $PROCS;
+    do
```

Comment:
I think typical bash style is to put `do` on the same line as `for` (similar to `then`).